docs/modelserving/v1beta1/transformer/feast/README.md

```diff
@@ -36,51 +183,88 @@ docker push {username}/driver-transformer:latest
 ```
 
 ## Create the InferenceService
-Please use the [YAML file](./driver_transformer.yaml) and update the `feast_serving_url` argument to create the `InferenceService`, which includes a Feast Transformer and a SKLearn Predictor.
-
 In the Feast Transformer image we packaged the driver transformer class so KServe knows to use the preprocess implementation to augment inputs with online features before making model inference requests.
 Then the `InferenceService` uses `SKLearn` to serve the [driver ranking model](https://github.com/feast-dev/feast-driver-ranking-tutorial), which is trained with Feast offline features, available in a gcs bucket specified under `storageUri`.
+Update the container's `image` field and the `feast_serving_url` argument to create the `InferenceService`, which includes a Feast Transformer and a SKLearn Predictor.
 
-```YAML
-apiVersion: "serving.kserve.io/v1beta1"
-kind: "InferenceService"
-metadata:
-  name: "sklearn-driver-transformer"
-spec:
-  transformer:
-    containers:
-    - image: kserve/driver-transformer:latest
-      name: driver-container
-      command:
-      - "python"
-      - "-m"
-      - "driver_transformer"
-      args:
-      - --feast_serving_url
-      - x.x.x.x:x
-      - --entity_ids
-      - driver_id
-      - --feature_refs
-      - driver_hourly_stats:acc_rate
-      - driver_hourly_stats:avg_daily_trips
-      - driver_hourly_stats:conv_rate
-  predictor:
-    sklearn:
-      storageUri: "gs://kfserving-examples/models/feast/driver"
-```
+=== "New Schema"
+    ```bash
+    cat <<EOF | kubectl apply -f -
+    apiVersion: "serving.kserve.io/v1beta1"
+    kind: "InferenceService"
+    metadata:
+      name: "sklearn-driver-transformer"
+    spec:
+      transformer:
+        containers:
+          - image: "{username}/driver-transformer:latest"
```

Comment:
We can update the kserve image?

Reply:
yes, we can update the image

Reply:
image is published `kserve/driver-transformer:latest`